Firestore/core/src/firebase/firestore/model/field_value.h

```diff
@@ -183,14 +151,12 @@ class FieldValue {
   static FieldValue FromGeoPoint(const GeoPoint& value);
   static FieldValue FromArray(const std::vector<FieldValue>& value);
   static FieldValue FromArray(std::vector<FieldValue>&& value);
-  static FieldValue FromMap(
-      const immutable::SortedMap<std::string, FieldValue>& value);
-  static FieldValue FromMap(
-      immutable::SortedMap<std::string, FieldValue>&& value);
 
   friend bool operator<(const FieldValue& lhs, const FieldValue& rhs);
 
  private:
+  friend ObjectValue;
```

Comment:
Shouldn't this be `friend class ObjectValue`?

Reply:
Done. (Incidentally, by fixing this up, we can additionally eliminate the forward declaration of ObjectValue. Also done.)